frontend/src/app/commonComponents/__test__/__snapshots__/ErrorPage.test.tsx.snap

```diff
@@ -28,15 +28,11 @@ exports[`ErrorPage displays the correct error message 1`] = `
           >
             <p
               class="usa-banner__header-text"
-            >
-              An official website of the United States government
-            </p>
+            />
```

Comment:
**question (non-blocking):** Should we init language for the snapshots? 🤔

Reply:
Ah, glad you noticed this! Using the translator function from the base `i18next` package rendered the text correctly in the app but didn't initialize for the snapshots. Addressed in 7ef22f7.